### ISSUE TITLE
Shooting a health pickup from the health pickups tutorial will throw an exception

### ISSUE THREAD

**Issue by jamiebrynes7** [spatialos/gdk-for-unity-fps-starter-project#212]
**Affects:** Release v0.2.4 and up (may affect earlier versions as well)
**Internal Ticket:** [UTY-2130](https://improbableio.atlassian.net/browse/UTY-2130)

---

#### Description

After following the [health packs tutorial](https://docs.improbable.io/unity/alpha/projects/fps/tutorial), shooting the health pickups results in an stack trace like the following:

```
ArgumentException: A component with type:Component has not been added to the entity.
Unity.Entities.EntityComponentStore.AssertEntityHasComponent (Unity.Entities.Entity entity, Unity.Entities.ComponentType componentType) (at Library/PackageCache/com.unity.entities@0.0.12-preview.33/Unity.Entities/EntityComponentStore.cs:488)
Unity.Entities.EntityComponentStore.AssertEntityHasComponent (Unity.Entities.Entity entity, System.Int32 componentType) (at Library/PackageCache/com.unity.entities@0.0.12-preview.33/Unity.Entities/EntityComponentStore.cs:494)
Unity.Entities.ComponentDataFromEntity`1[T].get_Item (Unity.Entities.Entity entity) (at Library/PackageCache/com.unity.entities@0.0.12-preview.33/Unity.Entities/Iterators/ComponentDataFromEntity.cs:116)
Improbable.Gdk.Guns.ServerShootingSystem.OnUpdate () (at Packages/com.improbable.gdk.guns/Systems/ServerShootingSystem.cs:44)
```

#### Workaround

Don't shoot the health pickups. 😛

**Comment by paulbalaji** [comment]
A quick-fix for this issue has been merged, and will be part of the next release.

**Comment by jamiebrynes7** [comment]
Released in [v0.2.6](https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.2.6)!
